src/peft/utils/save_and_load.py

```diff
@@ -200,6 +205,11 @@ def renamed_dora_weights(k):
             if any(f"{module_name}.modules_to_save.{adapter_name}" in key for module_name in model.modules_to_save):
                 to_return[key.replace("modules_to_save.", "")] = value
 
+    # ADDITIONAL TRAINING MODULES
+    for name, module in model.named_modules():
+        if isinstance(module, AuxiliaryTrainingWrapper):
```

Comment:
If `AuxiliaryTrainingWrapper` is a `ModulesToSaveWrapper`, is this not a duplication of the explicit `modules_to_save` handling directly above?

Reply:
Yes. I thought about explicitly excluding that case but I don't see a benefit. I wanted to discuss if we can drop the code before that and do all the `modules_to_save` handling in the additional training modules section. WDYT?

Reply:
Hmm, would the benefit not be that we avoid doing the same work twice? From that perspective, I'd say let's drop it. Alternatively, we can add a comment to drop it later if we plan to refactor to unify the trainable module handling.

Reply:
Dropped it.